### PR TITLE
BAH-1995 | patient-documents-display-control-ui-fixes-in-dashboard

### PR DESCRIPTION
**Pull request by shilpa-iplit**
patient-documents-display-control-ui-fixes-in-dashboard

### CODE REVIEW

**Review by N0-man — APPROVED** [review]

File: ui/app/common/obs/views/showObservation.html [hunk 1]
```diff
@@ -1,25 +1,40 @@
...
-        <span ng-hide="observation.isImageConcept()" ng-if="::(!observation.isImageConcept() || (observation.isImageConcept() && !print))" class="testUnderPanel left">
+        <span ng-show="observation.groupMembers.length!==0" ng-if="::(!observation.isImageConcept() || (observation.isImageConcept() && !print))" class="testUnderPanel left">
+            <label class="obs-label" ng-if="::configIsObservationForImages">{{::(observation.concept.shortName || observation.concept.name)}}</label>
+            <label ng-if="::!configIsObservationForImages">{{::(observation.concept.shortName || observation.concept.name)}}</label>
+            <hint concept-details="::{
+            lowNormal:observation.lowNormal||observation.concept.lowNormal,
+            hiNormal:observation.hiNormal||observation.concept.hiNormal,
+            }"/>
+            <a href="{{'/document_images/' + observation.value}}" download ng-if="observation.isVideoConcept()" class="button btn fr btn-download"><i class="fa fa-download" aria-hidden="true"></i></a>
```
Comment (gsluthra) [inline ui/app/common/obs/views/showObservation.html:10]: `/document_images/` is hardcoded. Isn't this a configurable value, and if someone chooses a diff config, won't this code break?

**Comment by gsluthra** [comment]
Please share some screenshots to show how things look now. It will be very helpful to understand the code changes.

File: ui/app/common/obs/views/showObservation.html (hunk 1 above)
Comment (shilpa-iplit) [reply ui/app/common/obs/views/showObservation.html:10]: In bahmni-india-package's openmrs container's volume 'bahmni-document-images:/home/bahmni/document_images' is mapped to patient-documents container's 'bahmni-document-images' volume.  Which is similar to bahmni installation's '/home/bahmni/document_images'

Lines from /usr/local/apache2/conf/bahmni-proxy.conf from proxy container:
#Patient Documents
ProxyPass /document_images http://patient-documents:80/document_images
ProxyPassReverse /document_images http://patient-documents:80/document_images

So I feel it is ok to map from /document_images
Please check. Thank you.

Comment (gsluthra) [reply ui/app/common/obs/views/showObservation.html:10]: @N0-man - What do you think? Is this something that might change deployment-to-deployment? For instance, when S3 is used, or when an on-prem installation is chosen? If so, then we will need to make the `/document_images/` path a configuration in UI code.

Comment (shilpa-iplit) [reply ui/app/common/obs/views/showObservation.html:10]: AllPatientDocument
patientDocument

Old files will follow: patientid-encounterName-uuid.ext (eg. 6-Patient-Document-706a448b-3f10-11e4-adec-0800271c1b75.png)
New ones will follow: patientid_encounterName_uuid__filename.ext (eg. 6-Patient-Document-706a448b-3f10-11e4-adec-0800271c1b75__doc1.png)

Comment (N0-man) [reply ui/app/common/obs/views/showObservation.html:10]: We want to keep the /document_images intact inorder to protect the documents behind authentication and authorization.

@gsluthra we have a container running nginx + JS `patient-documents` that is protecting the files behind `document_images` - see [this](https://github.com/BahmniIndiaDistro/helm-umbrella-chart/blob/c6c01e9421da1f5559c51d9d0897d18f8a8d6d97/templates/ingress.yaml#L56) and [this](https://github.com/Bahmni/patient-documents/blob/0acfbd837bb3598a00ce6707579ead6487f576d0/package/docker/default.conf.template#L20)

**Review by N0-man — APPROVED** [review]

**Pull request merged**